### PR TITLE
[Help Text] updated dashboard help text

### PR DESCRIPTION
**Pull request by llevitis**
This PR updates the Dashboard help text and gets rid of most HTML specific formatting, just preserving paragraph formatting and bold/italic formatting.

### CODE REVIEW

File: SQL/0000-00-04-Help.sql [hunk 1]
```diff
@@ -22,7 +22,7 @@ CREATE TABLE `help` (
...
... have been recruited for a given study.  Click the “Views” button in the right-hand corner for a breakdown by site. Note that charts include data from ineligible, excluded, and consent-withdrawn candidates.\r\n<b>Study Progression:</b> View a timeline of scans collected by each study site, or a timeline of candidate recruitment per site. Zoom in on the chart to view detailed data points. Note that charts include data from ineligible, excluded, and consent withdrawn candidates.\r\n<b>My Tasks:</b> User-specific list of outstanding tasks such as data entry conflicts, incomplete forms, or new and pending scans awaiting review. Click on any task panel to jump to the appropriate module.\r\n<b>Document Repository Notifications:</b> View recently uploaded documents for your study or site user’s study or site.  Click on a document name to download the file directly from the Dashboard. Click on “Document Repository” to view the complete list of available documents.\r\n<b>Behavioral Feedback Notifications:<\b> Track issues and corrections to behavioural/clinical data points. Click on a notification to redirect to the instrument with new changes or feedback. \r\n','2014-09-01 00:00:00',NULL),
```
Comment (ZainVirani) [inline SQL/0000-00-04-Help.sql]: `<b>Behavioral Feedback Notifications:<\b>`
This tag doesn't close properly and the entire line is bold

Comment (ZainVirani) [reply SQL/0000-00-04-Help.sql]: There should be better paragraph spacing as well

**Comment by llevitis** [comment]
image

updated version

**Comment by driusan** [comment]
Replaced by markdown text in #3148 

(The SQL script in this patch also has typos like missing the closing `>` for some HTML tags, so it wouldn't have worked anyways.)

**Pull request closed without merging**